src/test/java/teammates/test/cases/ui/browsertests/AdminSearchPageUiTest.java

```diff
@@ -74,6 +78,10 @@ private void testSearch() {
         assertTrue(isSearchPanelPresent());
         assertTrue(isSearchDataDisplayCorrect());
         
+        StudentAttributes student = testData.students.get("student1InCourse1");
+        InstructorAttributes instructor = testData.instructors.get("instructor1OfCourse1");
```

Comment:
Sure. I think there is quite a number of tests with the same problem, should I create an issue for all of them, or create a separate issue for each test?